Plugins/Public/base_plugin/ExportData.cpp

```diff
@@ -169,8 +169,12 @@ void ExportData::ToJSON()
 				item.write("is_public", i->second.is_public);
 				
 				const GoodInfo* gi = GoodList::find_by_id(i->first);
+				
 				wstring name = HkGetWStringFromIDS(gi->iIDSName);
-				item.write("item", wstos(HtmlEncode(name)).c_str());
+				item.write("name", wstos(HtmlEncode(name)).c_str());
```

Comment:
has occurred to me that we should not be doing HtmlEncode on this

Reply:
indeed. works without it